play-ahc-ws-standalone/src/test/scala/play/api/libs/ws/ahc/AhcWSRequestSpec.scala

```diff
@@ -414,7 +414,10 @@ class AhcWSRequestSpec
       withClient { client =>
         val req: AHCRequest = client
           .url("http://playframework.com/")
-          .withHttpHeaders(HttpHeaderNames.CONTENT_TYPE.toString() -> "application/x-www-form-urlencoded") // set content type by hand
+          .withHttpHeaders(
+            // set content type by hand
+            HttpHeaderNames.CONTENT_TYPE.toString() -> "application/x-www-form-urlencoded"
+          )
```

Comment:
New scalafmt rules may add breaklines on lines with a traling comment. The comment ends up in odd places. This change is 50% scalasteward (add newlines) and 50% manual (relocate the comment in a meaningful place).